src/inputs/labeled-inputs.scss

```diff
@@ -52,26 +53,37 @@
     font-size: $iui-font-size-small;
     line-height: $iui-line-height;
     margin-top: $iui-inputs-margin;
-
     @include themed {
       color: t(iui-text-color-muted);
     }
 
-    svg {
+    > svg,
+    :not(.iui-button) svg {
       width: $iui-icons-default;
       height: $iui-icons-default;
       margin-right: $iui-xs;
-
       @include themed {
         fill: t(iui-icons-color);
       }
     }
 
+    > .iui-borderless {
+      border-top-left-radius: 0;
+      border-bottom-left-radius: 0;
+    }
+
     a {
       user-select: none;
     }
   }
 
+  &:not(.iui-inline) > .iui-message > .iui-borderless {
```

Comment:
So this one is not needed then as you removed non inline example.

Reply:
Removed.